File: TransCoder_tokenized_test_set_functions/python/DELETE_CONSECUTIVE_WORDS_SEQUENCE_1.py
def removeConsecutiveSame ( v ) :
    st = [ ]
    for i in range ( len ( v ) ) :
        if ( len ( st ) == 0 ) :
            st . append ( v [ i ] )
        else :
            Str = st [ - 1 ]
            if ( Str == v [ i ] ) :
                st . pop ( )
            else :
                st . append ( v [ i ] )
    return len ( st )

File: TransCoder_tokenized_test_set_functions/java/DELETE_CONSECUTIVE_WORDS_SEQUENCE_1.java
static int removeConsecutiveSame ( Vector < String > v ) {
  Stack < String > st = new Stack < > ( );
  for ( int i = 0;
  i < v . size ( );
  i ++ ) {
    if ( st . empty ( ) ) st . push ( v . get ( i ) );
    else {
      String str = st . peek ( );
      if ( str . equals ( v . get ( i ) ) ) st . pop ( );
      else st . push ( v . get ( i ) );
    }
  }
  return st . size ( );
}